api/src/main/java/org/eclipse/microprofile/metrics/SimpleTimer.java

```diff
@@ -98,6 +98,12 @@
 
     @Override
     long getCount();
+
+    @Override
+    /**
+     * @return returns the cumulative recorded time durations in nanoseconds
```

Comment:
should make this javadoc and the one for getElapsedTime similar so people can tell it's the same number expressed two different ways